mirror/dependencies/pnda-rpm-package-dependencies.txt

```diff
@@ -30,13 +30,15 @@ MySQL-python-1.2.5-1.el7
 nginx
 nmap-ncat
 ntp-4.2.6p5
+patch
 python2-pip
 python34-devel
 python34-pip
 python-carbon-0.9.15-6.el7
 python-devel
 redhat-lsb
 redis-3.2.3-1.el7
+ruby-devel
```

Comment:
ruby-devel need to be part of deb dependencies

Reply:
Added